### PR TITLE
feat: introduce repeater chart

### PR DESCRIPTION
**Pull request by scavnickyj**
JIRA: F1-148

---

> [!IMPORTANT]
> Please, **don't forget to run `rush change`** for the commits that introduce **new features** 🙏

---

Refer to [documentation](https://github.com/gooddata/gooddata-ui-sdk/blob/master/dev_docs/continuous_integration.md) to see how to run checks and tests in the pull request. This is the list of the most used commands:

```
extended test - backstop
```

```
// Tiger platform
extended test - tiger-cypress - integrated
extended test - tiger-cypress - isolated
extended test - tiger-cypress - record
```

```
// Bear platform
extended test - cypress - integrated
extended test - cypress - isolated
extended test - cypress - record
```

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18210/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19809/ : SUCCESS in 11m 46s

**Comment by scavnickyj** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2594/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by kandl** [comment]
extended test - backstop

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/2595/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/18216/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/2047/ : SUCCESS in 25m 05s

File: libs/sdk-ui-charts/src/charts/repeater/Repeater.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import omit from "lodash/omit.js";
+import { CoreRepeater } from "./CoreRepeater.js";
+import { constructRepeaterBuckets, constructRepeaterDimensions } from "./internal/repeaterExecution.js";
+
+//
+// Public interface
+//
+
+/**
+ * @beta
```
Comment (ivanmjartan) [inline libs/sdk-ui-charts/src/charts/repeater/Repeater.tsx:24]: Describe interface pls

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/repeater/RepeaterDescriptor.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        component: {
+            importType: "named",
+            name: "Repeater",
+            package: "@gooddata/sdk-ui-charts",
+        },
+        insightToProps: getInsightToPropsConverter<IRepeaterProps>({
+            attribute: singleAttributeBucketConversion("attribute", BucketNames.ATTRIBUTE),
+            columns: multipleAttributesOrMeasuresBucketConversion("columns", BucketNames.COLUMNS),
+            filters: filtersInsightConversion("filters"),
+            // sortBy: sortsInsightConversion("sortBy"),
```
Comment (ivanmjartan) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/repeater/RepeaterDescriptor.ts:61]: remove it or implement it

**Comment by scavnickyj** [comment]
extended test - tiger-cypress - isolated

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Review by ivanmjartan — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4747

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/4006/ : SUCCESS in 13m 27s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19824/ : SUCCESS in 11m 19s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1664/ : SUCCESS in 14m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5176/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1885/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1675/ : SUCCESS in 16m 23s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/651/ : FAILURE in 8m 22s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4747

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4747

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/4008/ : SUCCESS in 16m 49s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/19827/ : SUCCESS in 11m 08s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/1666/ : SUCCESS in 11m 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/5178/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/1887/ : SUCCESS in 1m 32s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/1677/ : SUCCESS in 12m 28s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/653/ : FAILURE in 12m 10s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR4747

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**